components/style/values/computed/mod.rs

```diff
@@ -116,9 +116,9 @@ impl ToComputedValue for specified::Length {
             specified::Length::Absolute(length) => length,
             specified::Length::Calc(calc, range) => range.clamp(calc.to_computed_value(context).length()),
             specified::Length::FontRelative(length) =>
-                length.to_computed_value(context, /* use inherited */ false),
+                length.to_computed_value(context),
             specified::Length::ViewportPercentage(length) =>
-                length.to_computed_value(context.viewport_size()),
```

Comment:
Removed the additional arguments just to make this consistent with all the other `to_computed_value` calls.